### ISSUE TITLE
Absent cavum septum pellucidum

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7698]
**Preferred term label:**
Absent cavum septum pellucidum

**Synonyms**

**Definition (free text, please give PubMed ID)**
The cavum septum pellucidum (CSP) is an important fetal midline forebrain landmark that is normally visible by prenatal sonography. This term refers to the lack of visiblity of the CSP on prenatal sonography.
PMID:26123886

Comment:
The absence of the CSP during the prenatal period may signify additional underlying malformations such as malformations of the corpus callosum. Note that postnatally, the septum pellucidum normally is not hollow (cavum is Latin for hollow). See also Cavum septum pellucidum HP:0002389, which refers to the presence of a hollow septum pellucidum postnatally.

**Parent term (use hpo.jax.org/app)**
Abnormality of the septum pellucidum HP:0007375

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@sng2140 
Added as HP:4000138
Definition:
The cavum septum pellucidum (CSP) is an important fetal midline forebrain landmark that is normally visible by prenatal sonography. This term refers to the lack of visiblity of the CSP on prenatal sonography. The CSP normally can be visualized between 18 and 37 weeks of gestation. Lack of visualization of the CSP in this time period generally requires follow up and may indicate the presence of underlying malformations such as malformations of the corpus callosum. The lack of CSP before 18 weeks or after 37 weeks of gestation (or postnatally) is considered a normal finding.
